Source/VCGeneration/ConditionGeneration.cs

```diff
@@ -524,14 +525,16 @@ void ObjectInvariant()
         Contract.Invariant(cce.NonNullElements(vcResults));
       }
 
-      public readonly List<Counterexample> /*!>!*/ examples = new List<Counterexample>();
-      public readonly List<VCResult> /*!>!*/ vcResults = new List<VCResult>();
+      public readonly ConcurrentQueue<Counterexample> /*!>!*/
```

Comment:
Oof, this line cutoff is so narrow! What's the scenario in which we want 80 chars maximum?

Reply:
I often view code in an 80-character terminal, though I don't have a strong preference about that. But the main reason for this is that I originally wrote it as a diff on top of the code that existed before you joined that line. ;)

Reply:
Fixed by simplifying the initialization expressions to that they're on one <80 character line. :)

Reply:
You can make the RHS of these statements just `new ();`, which saves quite a few characters!

Reply:
Yep, that's what I did!